lib/capistrano/data_migrate.rb

```diff
@@ -0,0 +1,3 @@
+require 'capistrano'
```

Comment:
This file is not required by default. The user needs to manually add `require `capistrano/data_migrate` in their `Capfile`. I'm adding docs for this.